app/views/shared/_menu.html.erb

```diff
@@ -4,6 +4,7 @@
 
       <%= drawer_menu_navigation_item('Avance de Carrera', root_path) %>
       <%= drawer_menu_navigation_item('Todas las Materias', all_subjects_path) %>
+      <%= drawer_menu_navigation_item('Materias INCO semestre actual', current_optional_subjects_path) %>
```

Comment:
Should it be `Optativas INCO semestre actual` or something like that?

Reply:
Actually some subjects are required subjects not only optional like `Proyecto De Grado`